packages/wallets/solflare/src/adapter.ts

```diff
@@ -2,21 +2,25 @@ import type { WalletAdapterNetwork, WalletName } from '@solana/wallet-adapter-ba
 import {
     BaseMessageSignerWalletAdapter,
     isIosAndRedirectable,
+    isVersionedTransaction,
     scopePollingDetectionStrategy,
+    SendTransactionOptions,
     WalletConfigError,
     WalletConnectionError,
     WalletDisconnectedError,
     WalletDisconnectionError,
+    WalletError,
     WalletLoadError,
     WalletNotConnectedError,
     WalletNotReadyError,
     WalletPublicKeyError,
     WalletReadyState,
+    WalletSendTransactionError,
     WalletSignMessageError,
     WalletSignTransactionError,
 } from '@solana/wallet-adapter-base';
 import type { Transaction, TransactionVersion, VersionedTransaction } from '@solana/web3.js';
-import { PublicKey } from '@solana/web3.js';
+import { Connection, PublicKey, TransactionSignature } from '@solana/web3.js';
```

Comment:
```suggestion
import { type Connection, PublicKey, type TransactionSignature } from '@solana/web3.js';
```

Must be imported as types since they are only used as types. Run `pnpm run lint` to check.